index.js

```diff
@@ -0,0 +1,38 @@
+import ListBooks from './modules/App.js';
+import { DateTime } from './modules/luxon.js';
+
+const pDateTime = document.getElementById('dateTime');
+// pDateTime.textContent = "Hello"
```

Comment:
- [ ] Kindly remove all commented-out, un used/ unwanted pieces of code from your code base. It makes it look messy and cluttered.